contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,408 @@
+
+In the below example, we assign the user with `distinct_id` another ID: `alias_id`. This means that any events submitted using either `distinct_id` or `alias_id` will be associated with the same user.
+
+<MultiLanguage selector="tabs">
+
+```node
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'alias_id',
+})
+```
+
+```javascript
+posthog.alias('alias_id', 'distinct_id');
+```
+
+```python
+posthog.alias('distinct_id', 'alias_id', );
+```
+
+```php
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'alias_id'
+));
+```
+
+```ruby
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "alias_id",
+})
+```
+
+```go
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
+})
+```
+
+```java
+posthog.alias("distinct_id", "alias_id");
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "distinct_id": "distinct_id",
+        "alias": "alias_id"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "event": "$create_alias"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+There are two requirements when assigning an `alias_id`:
+
+1. It cannot be associated with more than one `distinct_id`.
+2. The `alias_id` **must not** be the `distinct_id` of any other user. For example:
+
+<MultiLanguage selector="tabs">
+
+```node
+// ❌ Not possible. 
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'distinct_id_of_another_user',
+})
+```
+
+```javascript
+// ❌ Not possible. 
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'distinct_id_of_another_user',
+})
+```
+
+```python
+# ❌ Not possible
+posthog.alias('distinct_id_of_another_user', 'distinct_id');
+```
+
+```php
+# ❌ Not possible
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'distinct_id_of_another_user'
+));
+```
+
+```ruby
+# ❌ Not possible
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "distinct_id_of_another_user",
+})
+```
+
+```go
+// ❌ Not possible
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "distinct_id_of_another_user",
+})
+```
+
+```java
+// ❌ Not possible
+posthog.alias("distinct_id", "distinct_id_of_another_user");
+```
+
+</MultiLanguage>
+
+If we encounter an `identify` or `alias` event with one of the above problems, the following will happen:
+
+1. We process the event normally (it will be ingested and show up in the UI)
+2. Merging users will be refused and an ingestion warning will be logged (see [ingestion warnings](/manual/data-management#ingestion-warnings) for more details).
+3. The event will be only be tied to the user associated with `distinct_id`
```

Comment:
this is true for when we encounter null etc invalid distinctIDs - the considerations and what will happen should be together and have this in the end. Move both to troubleshooting/FAQ section. into the I have dups section